OsmAnd/src/net/osmand/plus/views/layers/MapMarkersLayer.java

```diff
@@ -282,7 +282,7 @@ public void onPrepareBufferImage(Canvas canvas, RotatedTileBox tileBox, DrawSett
 		if (route != null && route.points.size() > 0) {
 			planRouteAttrs.updatePaints(app, nightMode, tileBox);
 			new Renderable.StandardTrack(new ArrayList<>(route.points), 17.2).
```

Comment:
PR should be reworked. Use code like [this](https://github.com/osmandapp/Osmand/blob/4e025d0ef6df0b52003b6aa39e6658b0a5dfa4a5/OsmAnd/src/net/osmand/plus/views/layers/GPXLayer.java#L1147) in case of route lines. You do not need change other classes.